tools/kbcheck/Cargo.toml

```diff
@@ -12,3 +12,5 @@ serde_json="1"
 serde_yaml="0.8"
 walkdir = "2"
 url={ version = "2", features = ["serde"] }
+thiserror = "1"
+clap = { version = "3.2.8", features = ["derive"] }
```

Comment:
We might have to pin this on a `3` or `3.2`. [There's a new release at least once per week](https://crates.io/crates/clap/versions), and manually keeping track of that might not be awesome.